app/models/job_invocation_api_composer.rb

```diff
@@ -0,0 +1,62 @@
+class JobInvocationApiComposer
+  attr_accessor :params, :job_invocation, :host_ids, :search_query
+
+  delegate :job_name, to: :job_invocation
```

Comment:
not sure if this is important or not but we use hash rocket style in whole Foreman AFAIK, we don't have to follow it but I just want to make sure that everyone is OK with this